### PR TITLE
add url to the list-view

### PR DESCRIPTION
**Pull request by Roland-Hufnagel**

### CODE REVIEW

**Comment by Roland-Hufnagel** [comment]
to have a look at it use:
https://capstone-project-git-feature-link-to-the-survey-roland-hufnagel.vercel.app/surveys

File: components/SurveyCard.js [hunk 1]
```diff
@@ -1,28 +1,66 @@
...
-      </Link>
-      <Link href={`/create/${id}`} passHref>
-        <StyledAnchor>Edit</StyledAnchor>
-      </Link>
-      <StyledButton onClick={onDelete}>Delete</StyledButton>
+      <h2>{title}</h2>
+      <time>{date}</time>
+      <DisplayLink>
+        <span>{url}</span>
+        <button
```
Comment (ThomasF85) [inline components/SurveyCard.js:13]: This button has no accessible label.

File: components/SurveyCard.js [hunk 2]
```diff
@@ -1,28 +1,66 @@
...
+        </Link>
+        <Link href={`/create/${id}`} passHref>
+          <StyledAnchor>Edit</StyledAnchor>
+        </Link>
+        <StyledButton onClick={onDelete}>Delete</StyledButton>
+      </ButtonContainer>
     </StyledSurveyCard>
   );
 }
+const ButtonContainer = styled.section``;
```
Comment (ThomasF85) [inline components/SurveyCard.js:33]: There is no need for a styled component that does not do anything.

File: components/SurveyCard.js [hunk 3]
```diff
@@ -1,28 +1,66 @@
...
+`;
 
 const StyledSurveyCard = styled.li`
   all: unset;
   border: 1px solid black;
   border-radius: 0.4em;
   padding: 0.5em;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
```
Comment (ThomasF85) [inline components/SurveyCard.js:59]: You are using em and px here for things that should probably scale proportionally. Better don't mix them in that case.

**Review by ThomasF85 — CHANGES_REQUESTED** [review]
Looks good. Just some minor changes requested.

File: components/SurveyCard.js (hunk 2 above)
Comment (Roland-Hufnagel) [reply components/SurveyCard.js:33]: now i gave it the styles for its children.

File: components/SurveyCard.js (hunk 3 above)
Comment (Roland-Hufnagel) [reply components/SurveyCard.js:59]: Thats right. Thanks a lot.

**Review by ThomasF85 — APPROVED** [review]
Looks good!

**Pull request merged**